workflow/lib/preprocess/preprocess.py

```diff
@@ -83,6 +85,93 @@ def extract_tile_metadata(
     return df
 
 
+def extract_well_metadata(
```

Comment:
I thought the plan was to not add well-specific functions to main, at least those that we would not expect a broad audience to use in the future. As part of our attempt to keep the main codebase clean and concise while having useful functionality.

Is my understanding of this off?